code/modules/modular_computers/hardware/hard_drive.dm

```diff
@@ -51,6 +51,7 @@
 	F.holder = src
 	stored_files.Add(F)
 	recalculate_size()
+	SEND_SIGNAL(F, COMSIG_MODULAR_COMPUTER_FILE_ADDED)
```

Comment:
Same as before, any chance of turning this signal into something like `COMSIG_MODULAR_COMPUTER_FILE_ADDING` and putting it right after the early return checks? Same for the other modular computer signal.

Reply:
b0f82e8247427939ed6cbf48e0ab546b3d354149